drivers/net/dm90x0.c

```diff
@@ -1554,8 +1554,7 @@ static int dm9x_addmac(FAR struct net_driver_s *dev, FAR const uint8_t *mac)
 
   /* Add the MAC address to the hardware multicast routing table */
 
-#warning "Multicast MAC support not implemented"
-  return OK;
+  return -ENOSYS;
```

Comment:
For these missing logic sections I'm fine with the change to put the correct return code (that seems more correct) but should we also have a comment near it indicating missing logic? That would also make it easier to find and implement these sections later?

Reply:
Done, change #warning to comment for reference.